pkg/networkservice/common/endpointtimeout/server.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package endpointtimeout provides server chain element that executes a callback when there were no connections for specified time
+package endpointtimeout
```

Comment:
Note: this can be used not only by endpoints. SDK chain elements can be included in any SDK bassed application.

Reply:
I can't come up with a better name. Could you suggest one?

Reply:
I would use just `timeout`, but we already have such element for connection timeout.

Reply:
We might call it: ```idle``` or ```onidle`` since what it is doing really is taking some action when idle.

Reply:
@d-uzlov Lets use `idle ` or `onidle` for name.